Update button style, create vertical scroll elements, add schedule

ericyc16 opened the pull request:
- On FAQ button hover, colors invert
- Add schedule
  - Change it to no longer be a modal window
- Add vertical scroll for FAQ & schedule when they become too large for screen height
  - When vertical scroll is needed and user scrolls down, horizontal scroll is temporarily disabled and vertical is temporarily enabled
  - When user has reached bottom of scrollable div, horizontal scroll is re-enabled and vertical is re-disabled
  - Similarly, when scrolling up, if div is not at top, horizontal scroll is temporarily disabled and vertical is temporarily enabled
  - Note this only occurs if user's mouse is over the scrollable div
    - If user's mouse is not over scrollable div, horizontal scroll continues as normal
- Change button/nav/modal color to better match new background

bluairis reviewed (changes requested):
On FAQ button hover, colors invert
- This looks good, I don't know how I feel about black boxes, but also I think that the color scheme matters less until later. 
- I think the buttons should be skinnier (width)
- I think nicole wants them to fit above the grassy knoll (the right one)

Add schedule 
- Change it to no longer be a modal window
> I can't tell if this is better or worse...I personally liked the modules but idk
- Add vertical scroll for FAQ & schedule when they become too large for screen height. When vertical scroll is needed and user scrolls down, horizontal scroll is temporarily disabled and vertical is temporarily enabled. When user has reached bottom of scrollable div, horizontal scroll is re-enabled and vertical is re-disabled. Similarly, when scrolling up, if div is not at top, horizontal scroll is temporarily disabled and vertical is temporarily enabled. Note this only occurs if user's mouse is over the scrollable div. If user's mouse is not over scrollable div, horizontal scroll continues as normal
> This works but it feels janky...maybe it is because the speed of the scroll is so fast that it feels like it is not working and also the literal second you hit the end the thing switches direction which is really catching me off guard
> It also somehow knocked off my members, sponsor and socials but idky

Change button/nav/modal color to better match new background
- this is good, and we might get more direction for this from design

ericyc16 commented:
> I don't know how I feel about black boxes

Changed it (& text color) to be a very dark purple, but we can continue to edit color scheme later.
Other FAQ button stuff has been addressed.

> I can't tell if this is better or worse...I personally liked the modules but idk

I changed it from modal since Anton seemed to prefer non-modal + the background Nicole gave showed a scroll thing. So maybe something to argue with them about

> This works but it feels janky...maybe it is because the speed of the scroll is so fast that it feels like it is not working and also the literal second you hit the end the thing switches direction which is really catching me off guard
It also somehow knocked off my members, sponsor and socials but idky

I changed the vertical scroll speed to be slower. Not sure how to fix the thing where when you hit the end it immediately switches direction, do you have any suggestions? Also I have no clue why it fucked with your stuff but should be fixed
